src/components/dashboardPanel/dashboardPanel.js

```diff
@@ -27,7 +27,9 @@ class DashboardPanel extends Component {
           </div>
           {this.props.actions ? <div className="panel-actions-container">{this.props.actions}</div> : ''}
         </div>
-        <div className="panel-contents">{this.props.children}</div>
+        <div className="panel-contents">
+        {this.props.children}
+        { this.props.initialIndicator ? <div className="initial-loading"><Spinner size='large'/></div> : null}</div>
```

Comment:
Return the panel-contents to its previous state and move the loading indicator to be a sibling element to the panel-contents.